documentation/oracle/governance/adding_a_price_identifier.md

```diff
@@ -0,0 +1,48 @@
+# Adding a Price Identifier
+
+## What is a price identifier?
+
+A [price identifier](../../synthetic_tokens/glossary.md) is a natural language descriptor of the reference index determining how much collateral is needed for a token sponsor to be properly collateralized. 
```

Comment:
Two comments:
1. nit: why not add the price identifier tag: `#price-identifier` to this link?
2. This definition of a price identifier seems fairly tied to the idea of a synthetic token. Should our language around DVM concepts be a little more generic?

Reply:
1. Yes, I was going for that and must have gotten so distracted just trying to get the right page to show up that I forgot about the anchor. 
2. I was hesitant to introduce concepts outside of what's in the glossary (although the second paragraph tries to do a bit of that). Happy to take suggestions you or Hart might have.

Reply:
2. SG -- just wanted to make sure it was intentional. Up to you and @hlambur how to message DVM generality.